networking/networking-operators-overview.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="networking-operators-overview"]
+= Networking operators overview
```

Comment:
You could capitalise "operators" in this title. Please see [this section of the guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization) regarding the capitalisation of "Operator".